vnstatui.go

```diff
@@ -26,27 +27,29 @@ var pagecontent = `<!DOCTYPE html>
 </head>
 <body>
   <img src="/img/vnstat-summary.png"/>
-  <img src="/img/vnstat-hourly.png"/>
   <img src="/img/vnstat-daily.png"/>
-  <img src="/img/vnstat-monthly.png"/>
```

Comment:
why droping monthly? Maybe we can keep it and make it optional? When I created vnstatui, the goal was mainly to track a monthly consumption